teamcity: failed test: TestDockerCLI

The following tests appear to have failed on master (acceptance): TestDockerCLI/test_disable_replication.tcl, TestDockerCLI

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestDockerCLI).

[#1485022](https://teamcity.cockroachdb.com/viewLog.html?buildId=1485022):


```
TestDockerCLI
--- FAIL: acceptance/TestDockerCLI (372.920s)

------- Stdout: -------
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI196157948
    test_log_scope.go:58: use -show-logs to present logs inline



TestDockerCLI/test_disable_replication.tcl
--- FAIL: acceptance/TestDockerCLI: TestDockerCLI/test_disable_replication.tcl (15.850s)





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cli/interactive_tests/common.tcl
+++ b/pkg/cli/interactive_tests/common.tcl
@@ -96,20 +96,21 @@ proc start_server {argv} {
 proc stop_server {argv} {
     report "BEGIN STOP SERVER"
     # Trigger a normal shutdown.
-    system "$argv quit"
-    # If after 5 seconds the server hasn't shut down, trigger an error.
-    system "for i in `seq 1 5`; do
+    # If after 30 seconds the server hasn't shut down, kill the process and trigger an error.
+    # Note: kill -CONT tests whether the PID exists (SIGCONT is a no-op for the process).
+    system "kill -TERM `cat server_pid` 2>/dev/null;
+            for i in `seq 1 30`; do
               kill -CONT `cat server_pid` 2>/dev/null || exit 0
               echo still waiting
               sleep 1
             done
             echo 'server still running?'
             # Send an unclean shutdown signal to trigger a stack trace dump.
-            kill -ABRT `cat server_pid`
+            kill -ABRT `cat server_pid` 2>/dev/null
             # Sleep to increase the probability that the stack trace actually
             # makes it to disk before we force-kill the process.
             sleep 1
-            kill -KILL `cat server_pid`
+            kill -KILL `cat server_pid` 2>/dev/null
             exit 1"
 
     report "END STOP SERVER"
@@ -131,13 +132,6 @@ proc flush_server_logs {} {
 
 proc force_stop_server {argv} {
     report "BEGIN FORCE STOP SERVER"
-    system "$argv quit & sleep 1
-            if kill -CONT `cat server_pid` 2>/dev/null; then
-              kill -TERM `cat server_pid`
-              sleep 1
-              if kill -CONT `cat server_pid` 2>/dev/null; then
-                kill -KILL `cat server_pid`
-              fi
-            fi"
+    system "kill -KILL `cat server_pid`"
     report "END FORCE STOP SERVER"
 }

--- a/pkg/cli/interactive_tests/test_log_config_msg.tcl
+++ b/pkg/cli/interactive_tests/test_log_config_msg.tcl
@@ -6,11 +6,14 @@ source [file join [file dirname $argv0] common.tcl]
 # flushed and not in the middle of a rotation.
 
 start_server $argv
-stop_server $argv
 
 start_test "Check that the cluster ID is reported at the start of the first log file."
-system "grep -qF '\[config\] clusterID:' logs/db/logs/cockroach.log"
+spawn tail -n 1000 -F logs/db/logs/cockroach.log
+eexpect "\\\[config\\\] clusterID:"
+eexpect "node startup completed"
 end_test
+
+stop_server $argv
 
 
 # Make a server with a tiny log buffer so as to force frequent log rotation.

